### PR TITLE
Package heptagon.1.05.00

### PR DESCRIPTION
**Pull request by gdelaval**
### `heptagon.1.05.00`
Compiler for the Heptagon/BZR synchronous programming language
Heptagon/BZR is a synchronous dataflow language whose syntax and
semantics is inspired from Lustre, with a syntax allowing the
expression of control structures (e.g., switch or mode automata).
Heptagon/BZR is a research compiler, whose aim is to facilitate
experimentation. The current version of the compiler includes the
following features:
- Inclusion of discrete controller synthesis within the compilation
- Expression and compilation of array values with modular memory optimization
See https://gitlab.inria.fr/synchrone/heptagon for further informations.

---
* Homepage: https://gitlab.inria.fr/synchrone/heptagon
* Source repo: git+https://gitlab.inria.fr/synchrone/heptagon.git
* Bug tracker: https://gitlab.inria.fr/synchrone/heptagon/-/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

**Comment by mseri** [comment]
Why is this replacing an already released package?

**Comment by gdelaval** [comment]
> Why is this replacing an already released package?

Because the repository URL has changed. The old URL is no longer accessible.

File: packages/heptagon/heptagon.1.05.00/opam [hunk 1]
```diff
@@ -22,21 +34,12 @@ depopts: [
...
-experimentation. The current version of the compiler includes the
-following features:
-- Inclusion of discrete controller synthesis within the compilation
-- Expression and compilation of array values with modular memory optimization
-See http://heptagon.gforge.inria.fr for further informations."""
-flags: light-uninstall
-extra-files: ["heptagon.install" "md5=eae4ae21fb60b00f2076c1252c1750c2"]
 url {
   src:
-    "https://github.com/ocaml/opam-source-archives/raw/main/heptagon-1.05.00.tar.gz"
```
[the remark this answers is not in the document]
Comment (mseri) [reply packages/heptagon/heptagon.1.05.00/opam:40]: If you want, the new tarball could be released as version `1.05.00-1` on the opam-repository, to make it clear that then contents are different but functionally it is the same package

[the remark this answers is not in the document]
Comment (gdelaval) [reply packages/heptagon/heptagon.1.05.00/opam:40]: I think I am ok with your first suggestion (take the archive from the cache), do I have to do something for that (and what?)?

[the remark this answers is not in the document]
Comment (mseri) [reply packages/heptagon/heptagon.1.05.00/opam:40]: Nothing, just revert your changes to the url, extra files and the extra file. Or we can close this PR and I cherrypick them by hand tomorrow

**Comment by mseri** [comment]
Metadata changes are picked in https://github.com/ocaml/opam-repository/pull/21425

**Pull request closed without merging**